src/app/(sidebar)/transaction/submit/page.tsx

```diff
@@ -67,6 +67,30 @@ const SUBMIT_OPTIONS = [
   },
 ];
 
+// traverse the xdr to json string and check if
+// it contains a soroban operation
+const isSorobanXdr = (xdrJsonString: string) => {
```

Comment:
I couldn't find other way. Do you have a suggestion other than traversing through the `xdrJsonString`?

Reply:
I can't think of anything else we could try.